drivers/storage/portworx/util/util.go

```diff
@@ -1770,3 +1780,80 @@ func AppendUserVolumeMounts(
 		}
 	}
 }
+
+func IsSupportedOCPVersion(k8sClient client.Client, targetVersion string) (bool, error) {
+	gvk := schema.GroupVersionKind{
+		Kind:    ClusterOperatorKind,
+		Version: ClusterOperatorVersion,
+	}
+
+	exists, err := coreops.Instance().ResourceExists(gvk)
+	if err != nil {
+		return false, err
+	}
+
+	if exists {
+		operator := &ocpconfig.ClusterOperator{}
+		err := k8sClient.Get(
+			context.TODO(),
+			types.NamespacedName{
+				Name: OpenshiftAPIServer,
+			},
+			operator,
+		)
+
+		if err != nil {
+			if errors.IsNotFound(err) {
+				return false, err
```

Comment:
We should not return error if the object is not found. It is an indicator that this is not a supported openshift environment.